src/components/Home/Home.js

```diff
@@ -67,7 +68,9 @@ const Home = () => {
           data-testid="section-additional-collection-gallery"
         >
           <div className="section-top contain-1440">
-            <p className="subhead"> {keyword} Collections</p>
+            <p className="subhead">
+              {keyword.charAt(0).toUpperCase() + keyword.slice(1)} Collections
```

Comment:
If you wanted to handle this via CSS, think you could add a style something like: `text-transform: capitalize;`

Reply:
yeah. will do. I was hoping I was missing some built in way to do this.